docs/tutorials/reading-list.md

```diff
@@ -126,6 +126,10 @@ Some titles have been shortened for organizational purposes.
 
 [Java Tutorials for Beginners]: https://www.geeksforgeeks.org/java/
 
+*Tutorial* - [Learn Java from scratch][]
+
+[Learn Java from scratch]: https://www.rcsdk12.org/cms/lib/NY01001156/Centricity/Domain/4951/Head_First_Java_Second_Edition.pdf
```

Comment:
Thanks for your interest, but is there a compelling reason to recommend a 15 year old book that even predates Android, over the plethora of newer resources? And is this link even supposed to be public?

Reply:
Your'e right the book is older but content in the book is nice in my point of view ans so I suggest it to all. When I search for this book I find that link and also I don't know know much about that?
If you any further suggestions let me know?